tests/com/tony/server/ArgumentParserTest.java

```diff
@@ -1,41 +1,45 @@
 package com.tony.server;
 
+import org.junit.Before;
 import org.junit.Test;
 
 import static org.junit.Assert.*;
```

Comment:
Don't use wildcard imports, it's better to be explicit with what you are bringing in.

If you don't like the longer list of imports at the top: http://vim.wikia.com/wiki/Folding